### PR TITLE
Fix instange groups docs

### PR DESCRIPTION
**Pull request by klaus993**
This addresses issue #4404, fixes instance groups docs given that the command it indicates there doesn't exactly work, and modifies an example in the command's `--help` UX.

### CODE REVIEW

**Comment by klaus993** [comment]
Sorry, wrong command :)

**Comment by mikesplain** [comment]
Hi @klaus993 thanks for the contribution! This looks pretty good but you'll need to run `make gen-cli-docs` to update the docs for this as well. Mind checking those in and squashing?

/ok-to-test

**Comment by klaus993** [comment]
@mikesplain OK, sure!

**Comment by klaus993** [comment]
@mikesplain Done.

**Comment by klaus993** [comment]
I added another commit adding some clarification in the "Deleting an instance group" section of the same doc. I found that it disrupted my flow (by deleting the ASG automatically and not waiting for my `terraform apply`) and I though it might best be clarified. I explained it also in the commit message. What do you think?

**Comment by mikesplain** [comment]
This is great! Thanks so much for the help @klaus993! 

/lgtm

**Comment by klaus993** [comment]
Sorry, I later realised that the last clarification about instancegroup deletion may also be included in the `--help` UX, so to end with it now (and not submit another PR later (my TOC is great)) I cancel lgtm and push this other commit.

**Comment by klaus993** [comment]
@mikesplain Sorry, now it's ready. PTAL and give me your comments or your lgtm :)

File: docs/instance_groups.md [hunk 1]
```diff
@@ -200,8 +203,7 @@ If you decide you don't need an InstanceGroup any more, you delete it using: `ko
 
 Example: `kops delete ig morenodes`
 
-No rolling-update is needed (and note this is not currently graceful, so there may be interruptions to
-workloads where the pods are running on those nodes).
+No `kops update cluster` nor `kops rolling-update` is needed, so **be careful** when deleting an instance group, your nodes will be deleted automatically (and note this is not currently graceful, so there may be interruptions to workloads where the pods are running on those nodes).
```
Comment (mikesplain) [inline docs/instance_groups.md]: Typo `no`

Comment (mikesplain) [reply docs/instance_groups.md]: Also, I would turn the content of the () to be a fresh sentence to make it easier to read.

**Comment by chrislovecnm** [comment]
@mikesplain I already approved it, will let you merge with a lgtm

**Comment by mikesplain** [comment]
That’s weird @chrislovecm, it changed after you approved. Either way, that’s for the contribution @klaus993! We can fix in other prs!

/lgtm

File: docs/instance_groups.md (hunk 1 above)
Comment (klaus993) [reply docs/instance_groups.md]: @mikesplain Could you explain the typo? I don't get it

**Comment by klaus993** [comment]
You're welcome! Glad to contribute.

File: docs/instance_groups.md (hunk 1 above)
Comment (mikesplain) [reply docs/instance_groups.md]: Nevermind! My mistake @klaus993!

Comment (klaus993) [reply docs/instance_groups.md]: OK! Great! About the thing between parenthesis, these days I will maybe submit another PR to leave it all clear and understandable :)

**Pull request merged**